tests/app/conftest.py

```diff
@@ -29,7 +23,7 @@ async def test_init_db() -> AsyncGenerator[None, None]:
     await close_db()
 
 
-@pytest_asyncio.fixture
+@pytest.fixture
```

Comment:
is there a reason asyncio was removed here, but not from the above `test_init_db` method?

Reply:
Yeah, I tried to remove it but it was mandatory due to the `loop_scope` parameter that is not available to `pytest.fixture`.